docs/examples/logs/example.py

```diff
@@ -0,0 +1,46 @@
+import logging
+
+from opentelemetry import trace
+from opentelemetry.exporter.otlp.proto.grpc.log_exporter import OTLPLogExporter
+from opentelemetry.sdk.logs import OTLPHandler, get_log_emitter_provider
+from opentelemetry.sdk.logs.export import SimpleLogProcessor
+from opentelemetry.sdk.trace import TracerProvider
+from opentelemetry.sdk.trace.export import (
+    ConsoleSpanExporter,
+    SimpleSpanProcessor,
+)
+
+trace.set_tracer_provider(TracerProvider())
```

Comment:
Curious, what is the purpose of adding the tracing pipeline in this example? Will there be an example utilizing the [logging instrumentation](https://github.com/open-telemetry/opentelemetry-python-contrib/tree/main/instrumentation/opentelemetry-instrumentation-logging) to demonstrate correlation?

Reply:
Yes, this is added for trace context correlation. We don't need instrumentation anymore if we have SDK setup.

Reply:
I might be missing something but I can't see trace context information in the sample output?

Reply:
It's because of the exporter used here. Only part of the log record is shown using logging exporter, please look at [this](https://github.com/open-telemetry/opentelemetry-collector/blob/b6ef0d6fd49795d7619383698388cea6bad3da59/internal/otlptext/logs.go#L29) and [this](https://github.com/open-telemetry/opentelemetry-collector/blob/b6ef0d6fd49795d7619383698388cea6bad3da59/internal/otlptext/databuffer.go#L154) from collector.